lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -103,6 +104,8 @@ def instructor_dashboard_2(request, course_id):
 
     is_white_label = CourseMode.is_white_label(course_key)
 
+    reports_enabled = configuration_helpers.get_value('show_ecommerce_reports', False)
```

Comment:
Let's use ALL CAPS for any new settings we add, so this would be `SHOW_ECOMMERCE_REPORTS`.